ls1intum/Artemis
# Fix position of ssh button

### Checklist

### Motivation and Context


Fix one more styling issue introduced by the Artemis version 5.0.0

### Description

Fixes the positioning of the "use SSH" button when cloning a programming exercise. (see screenshot)

### Steps for Testing


1. Log in to Artemis
2. Navigate to a programming exercise
3. Try to clone it and check the placement of the button

### Screenshots



![Bildschirmfoto 2021-06-30 um 22 24 27](https://user-images.githubusercontent.com/38322605/124026430-f70b7100-d9f1-11eb-840d-d81a10b78d20.png)
